src/components/SpreadsheetUploader/index.tsx

```diff
@@ -13,14 +13,26 @@ const acceptedTypes = [
   'text/csv',
 ];
 
-const displayCell = (cellData) => {
+const displayCell = (cellData: any) => {
```

Comment:
Better to use `unknown` here:
```suggestion
const displayCell = (cellData: unknown) => {
```